jupyter-lib/kotlin-jupyter-api/src/main/kotlin/org/jetbrains/kotlin/jupyter/api/api.kt

```diff
@@ -0,0 +1,74 @@
+package org.jetbrains.kotlin.jupyter.api
+
+/**
+ * Single evaluated notebook cell representation
+ */
+interface CodeCell {
+    /**
+     * Displayed cell ID
+     */
+    val id: Int
+
+    /**
+     * Internal cell ID which is used to generate internal class names and result fields
+     */
+    val internalId: Int
+
+    /**
+     * Cell code
+     */
+    val code: String
+
+    /**
+     * Cell code after magic preprocessing
+     */
+    val preprocessedCode: String
+
+    /**
+     * Cell result value
+     */
+    val result: Any?
+
+    /**
+     * Cell standard output
+     */
+    val streamOutput: String
+
+    /**
+     * Previously evaluated cell
+     */
+    val prevCell: CodeCell?
+}
+
+interface KotlinKernelHost {
```

Comment:
Javadoc is missing for this one.

Reply:
Added Javadoc, also added `addLibrary` method